src/analytics/Properties.tsx

```diff
@@ -67,6 +67,7 @@ import { NftOrigin } from 'src/nfts/types'
 import { NotificationReceiveState } from 'src/notifications/types'
 import { AdventureCardName } from 'src/onboarding/types'
 import { RecipientType } from 'src/recipients/recipient'
+import { QrCode } from 'src/send/actions'
```

Comment:
this is probably fine as is but would be nice if this is on send/types so there isn't any cyclic imports.